README.md

```diff
@@ -95,6 +95,7 @@ the COOL environment.
 | cert_bucket_name | The name of the AWS S3 bucket where certificates are stored. | `string` | `cisa-cool-certificates` | no |
 | cool_domain | The domain where the COOL resources reside (e.g. "cool.cyber.dhs.gov"). | `string` | `cool.cyber.dhs.gov` | no |
 | dns_ttl | The TTL value to use for Route53 DNS records (e.g. 86400).  A smaller value may be useful when the DNS records are changing often, for example when testing. | `number` | `60` | no |
+| email_sending_domain | The domain to send emails from within the assessment environment (e.g. "example.com"). | `string` | `example.com` | no |
```

Comment:
Do you mean the domain or the hostname?  If the user wants to send email from `treeofwoe.thulsa-doom.com` is that the value they specify for this variable?  Or do they just specify the domain `thulsa-doom.com` and the `treeofwoe` is specified elsewhere?

Reply:
If the user is supposed to specify the hostname then I suggest changing the name of the variable to make this clearer.  At a minimum, you should change the default to `mail.example.com` if the user is expected to specify the hostname instead of just the domain.

Reply:
Honestly, I don't know how much this value really matters.  If you set it to `thulsa-doom.com`, you can successfully send mail from either `conan@thulsa-doom.com` or `conan@treeofwoe.thulsa-doom.com`.  I did a test of this and it looks like as long as there is a valid MX record for `thulsa-doom.com` and `treeofwoe.thulsa-doom.com`, either one should be acceptable.

That is why I worded it as `domain` instead of `hostname` or `fully-qualified domain name`.

PS: I used https://mail-tester.com to get a score (with explanations) for the test emails I sent and they were scored the same, regardless of whether they were sent from `conan@thulsa-doom.com` or `conan@treeofwoe.thulsa-doom.com`.  For these tests, my `email_sending_domain` was just the `thulsa-doom.com` domain (not really, but the domain that I have been using for my testing).

Reply:
Perfect.  I withdraw the question.